extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -373,40 +369,42 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
   /**
-   * Reads the storyNextUp param and sets it as the auto-advance-after attribute
-   * if this is the last page and there isn't a value set by the publisher.
+   * Reads the storyNextUp param if provided and sets the auto-advance-after
+   * attribute if there isn't a value set by the publisher. The
+   * auto-advance-after attribute will be set to the tagName of the first
+   * video if there is one, or the value of the storyNextUp param otherwise.
    * @private
    */
   maybeSetStoryNextUp_() {
     const autoAdvanceAttr = this.element.getAttribute('auto-advance-after');
     const storyNextUpParam = Services.viewerForDoc(this.element).getParam(
       'storyNextUp'
     );
-    if (
-      autoAdvanceAttr === null &&
-      storyNextUpParam !== null &&
-      this.isLastPage_()
-    ) {
-      addAttributesToElement(
-        this.element,
-        dict({'auto-advance-after': storyNextUpParam})
-      );
+    if (autoAdvanceAttr === null && storyNextUpParam !== null) {
+      const video = this.getFirstVideo_();
+      if (video !== null) {
+        addAttributesToElement(
+          this.element,
+          dict({'auto-advance-after': video.tagName})
```

Comment:
This should be `video.id` instead of `video.tagName`.

Reply:
Done. Aha makes sense. Thanks!